docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,3685 @@
+
+Explaining Keras text classifier predictions with Grad-CAM
+==========================================================
+
+We can use ELI5 to explain text-based classifiers, i.e. models that take
+in a text and assign it to some class. Common examples include sentiment
+classification, labelling into categories, etc.
+
+The underlying method used is 'Grad-CAM'
+(https://arxiv.org/abs/1610.02391). This technique shows what parts of
+the input are the most important to the predicted result, by overlaying
+a "heatmap" over the original input.
+
+See also the tutorial for images
+(https://eli5.readthedocs.io/en/latest/tutorials/keras-image-classifiers.html).
+Certain sections such as 'removing softmax' and 'comparing different
+models' are relevant for text as well.
+
+Set up
+------
+
+First some imports
```

Comment:
Would be nice to state which tensorflow and keras versions this is tested with, so that in the future it's possible to reproduce it.